src/cloud/benchflow/sender/sender.go

```diff
@@ -379,27 +431,35 @@ func main() {
 	kafkaIp = viper.GetString("kafka_host")
 	kafkaPort = viper.GetString("kafka_port")
 	cassandraHost = viper.GetString("cassandra_host")
+	driverMemory = viper.GetString("driver_memory")
+	executorMemory = viper.GetString("executor_memory")
 	minioHost = viper.GetString("minio_host")
-	minio_port = viper.GetString("minio_port")
+	minioPort = viper.GetString("minio_port")
+	minioAccessKey = viper.GetString("minio_access_key")
+	minioSecretKey = viper.GetString("minio_secret_key")
+	runsBucket = viper.GetString("runs_bucket")
 	sparkHome = viper.GetString("spark_home")
 	sparkMaster = viper.GetString("spark_master")
-	spark_port = viper.GetString("spark_port")
+	sparkPort = viper.GetString("spark_port")
 	alluxio_port = viper.GetString("alluxio_port")
 	pysparkCassandraVersion = viper.GetString("pyspark_cassandra_version")
 	analysersPath = viper.GetString("analysers_path")
 	transformersPath = viper.GetString("transformers_path")
 	configurationsPath = viper.GetString("configurations_path")
+	// TODO: Take from the right file on Minio
```

Comment:
@simonedavico As discussed in https://github.com/benchflow/drivers-maker/pull/27#discussion_r66873907, we need to remove some information on the environment variables passed to `collectors` and `monitors`.  Anyway, @Cerfoglg you should use the `BENCHFLOW_EXPERIMENT_ID` or the `BENCHFLOW_TRIAL_ID ` to access the `yml` file simply by replacing `.` with `/` that is something we can hardcode since it is an assumption we anyway have for `Minio` keys.